pinot-plugins/pinot-file-system/pinot-s3/pom.xml

```diff
@@ -35,10 +35,10 @@
   <url>https://pinot.apache.org</url>
   <properties>
     <pinot.root>${basedir}/../../..</pinot.root>
-    <http.client.version>4.5.14</http.client.version>
```

Comment:
Since the dependency is already declared in the root pom, we shouldn't need to override it again. Ideally we shouldn't add extra exclusions to the dependency in the child pom as well